topics/transcriptomics/tutorials/pinpoint-key-pathways-with-heinz/tutorial.md

```diff
@@ -0,0 +1,404 @@
+> >   **Build Dataset List** from the dropdown menu.
+> >
+> > 3. In the next dialog window, you need to give a name, here we just set it to `CP`, then click **Create list**.
+> >   ![List of suggested paired datasets](../../images/create_collection.png)
+> >
+> > 4. Hidden these selected files by clicking on **for all selected..** and selecting **Hidden datasets**.
+> >   **Note:** This step is optional, we do it here to keep Galaxy history clean.
+> >
+> > 5. Redo the Step 2, 3, 4 for CN, set the name of the data list as 'CN'.
+> {: .details}
+{: .tip}
+
+
+
+# Differential Expression Analysis (DEA) by DESeq2
+
+## What is differential expression analysis?
+
+> ### {% icon comment %} A defintion
+>
+> The definition of differential expression analysis given by
+> [EBI](https://www.ebi.ac.uk/training/online/course/functional-genomics-ii-common-technologies-and-data-analysis-methods/differential-gene)
+> means taking the normalised read count data and performing statistical analysis to discover quantitative
+> changes in expression levels between experimental groups. For example, we use statistical testing to decide
+> whether, for a given gene, an observed difference in read counts is significant, that is, whether it is greater
+> than what would be expected just due to natural random variation.
+>
+{: .comment}
+
+In principle, DEA is a causal analysis; but in reality, it is hampered by the complexity of the experimental
+situation and measurement. Back to our datasets, CP and CN, they are from two experimental groups. By DEA,
+we hope to pinpoint the candidate genes relevant to dental caries first, then we will use Heinz to infer the
+related pathways.
+
+## Which tools are available for DEA?
+
+There are a few canned tools commonly used for DEA, like [Limma](https://bioconductor.org/packages/release/bioc/html/limma.html)
+and [DESeq2](https://bioconductor.org/packages/release/bioc/html/DESeq2.html). If you are interested, you
+may look up the pros and cons of each tool. Here we use DESeq2.
+
+
+## Perform Differential Expression Analysis (DEA)
+
+After learning about differential expression analysis, let's get some hands-on experience.
+
+> ### {% icon hands_on %} Hands-on: DEA via DESeq2
+>
+> - **DESeq2** {% icon tool %} with the following parameters
+>   - *"Specify a factor name"*: `dental_caries` (under `1: Factor`)
+>   - *"Specify a factor level"*: to `CP` (under `1: Factor level`)
+>       - {% icon param-files %} *"Counts file(s)"*: `CP` (collection)
+>   - *"Specify a factor level"*: `CN` (under `2: Factor level`)
+>       - {% icon param-files %} *"Counts file(s)"*: `CN` (collection)
+>   - *"Files have header"*: `No`
+>   - *"Visualising the analysis results"*: `No`
+>   - Leave all other parameters to the default settings <br><br>
+>
+{: .hands_on}
+
+It usually takes 10 - 15 minutes to finish DESseq2. After the analysis is finished, have a look at the file, it should look like something this:
```

Comment:
did you mean it will be 10-15 minutes on the training data? or in general? if the former that might be a bit long for a tutorial, if the latter, maybe add a sentence like "but since we are using small datasets for this example, it will be much quicker" so they know they won't have to wait that long

Reply:
For the demo data, it still takes about 10 minutes or something. Would it be a big issue?

Reply:
not a big issue, but might be nice to give them some background reading or questions orso for while they wait.

P.S. when I just ran it on the EU server it finished in 2 minutes, so it should be fine